API-proposed.yaml

```diff
@@ -2461,16 +2463,21 @@ components:
     XYModel:
       required:
       - series
+      - style
       - title
       type: object
       properties:
         series:
           type: array
+          description: The collection of series
           items:
             $ref: '#/components/schemas/SeriesModel'
+        style:
+          $ref: '#/components/schemas/OutputElementStyle'
```

Comment:
I think the style element should be in the SeriesModel and not globally across all the series, for example, for series color, symbol or line style. 

We could decide to keep a global style here for common styles and the SeriesModel would have optionally additional or overwriting styles in it.

Reply:
Ack, and this comment currently applies to the back-end counterpart; will check this.